Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/OCRWorkflow.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.data.database.beans;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "ocrworkflow")
+public class OCRWorkflow extends BaseBean {
```

Comment:
The term _workflow_ is already in use in Production with a different meaning. Maybe we can find a not-yet-used name for this class, like _OCR configuration_ (`OcrConfig`).

Reply:
That ambiguity is [well known](https://ocr-d.de/en/spec/glossary#ocr-d-workflow) and oft discussed. So far, I have never seen a good counter proposal. _OCR configuration_ gets closest indeed – I could live with that.

Within [ocrd_kitodo](https://github.com/slub/ocrd_kitodo/) documentation, though, we decided to use _workflow_ and always disambiguate between (organisational) Kitodo Workflow and (technical) OCR Workflow.

Reply:
Agreed, _configuration_ may not be so good after all. Yes, _OCR profile_ would also be acceptable, if properly explained/documented, as indeed would have to be done in any case.